src/test/lib/storage/table_test.cpp

```diff
@@ -137,13 +137,13 @@ TEST_F(StorageTableTest, FillingUpAChunkFinalizesIt) {
 
   const auto c = t->get_chunk(ChunkID{0});
   auto mvcc_data = c->mvcc_data();
-  EXPECT_FALSE(mvcc_data->max_begin_cid);
+  EXPECT_EQ(mvcc_data->max_begin_cid.load(), MvccData::MAX_COMMIT_ID);
   EXPECT_TRUE(c->is_mutable());
 
   t->append({6, "world"});
```

Comment:
Rename `t` to `table`? I'll pay for the storage costs.

Reply:
That's five bucks then.